Extensions/Console.json

```diff
@@ -0,0 +1,289 @@
+{
+  "author": "Arthur Pacaud (arthuro555)",
+  "description": "A console behavior that allows parsing simple commands inputed via a Text Input object.",
```

Comment:
Could you use this description for the short description? And maybe add a bit more details to this description?

Reply:
Text Entry, not Text Input.
(A text input would be a widget)